app/src/main/java/ar/com/wolox/android/example/ui/login/LoginFragment.java

```diff
@@ -20,6 +23,48 @@ public int layout() {
 
     @Override
     public void init() {
+        LoginPresenter loginPresenter = new LoginPresenter();
+        loginPresenter.attachView(this);
+
+        buttonLogin = getActivity().findViewById(R.id.button_login);
+        buttonSignup = getActivity().findViewById(R.id.button_signup);
+        textEmail = getActivity().findViewById(R.id.text_mail);
+        textPass = getActivity().findViewById(R.id.text_pass);
+
+        loginPresenter.getCredentials(getActivity());
```

Comment:
Remember the view doesn't request the presenter anything, it only inform events. Here it could be `onInit()`.